gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AdbTask.groovy

```diff
@@ -70,7 +110,7 @@ public class AdbTask extends org.gradle.api.DefaultTask {
             it.startsWith("$propertyKey:")
         }
         if (output == null) {
-            throw new IllegalStateException("Could not read property '$propertyKey' of $apkPath")
+            throw new IllegalArgumentException("Could not read property '$propertyKey' of $apkPath")
```

Comment:
Ah, okay, I see. 
I wonder whether we should better just return `null` instead. It does not seem like an exceptional state after all. 
Could then just use `?.` in line 31.
What do you think?